cmd/app/command/start.go

```diff
@@ -29,8 +30,10 @@ func NewCmdStart() *cobra.Command {
 	getCmd := &cobra.Command{
 		Use:   "start",
 		Short: "Node disk controller",
-		Long: ` watches for ndm custom resources via "ndmctl start" command `,
+		Long:  ` watches for ndm custom resources via "ndmctl start" command `,
```

Comment:
revert this line, you have some conflict here with recently merged change as ndmctl has been changed to ndm.